### PR TITLE
Remove Travis CI

### PR DESCRIPTION
**Pull request by andrewbranch**
It’s been failing without explanation for three days and we already have redundancy with DevOps and GitHub Actions.

### CODE REVIEW

File: src/testRunner/runner.ts [hunk 1]
```diff
@@ -209,12 +209,6 @@ namespace Harness {
             runners.push(new FourSlashRunner(FourSlash.FourSlashTestType.ShimsWithPreprocess));
             runners.push(new FourSlashRunner(FourSlash.FourSlashTestType.Server));
             // runners.push(new GeneratedFourslashRunner());
-
-            // CRON-only tests
-            if (process.env.TRAVIS_EVENT_TYPE === "cron") {
```
Comment (weswigham) [inline src/testRunner/runner.ts:214]: You should keep this. Yes, I know it says "TRAVIS_EVENT_TYPE", but we replicated the environment variable, verbatim, into our cron-based devops jobs :V

**Review by weswigham — COMMENTED** [review]
When I saw the unexplainable failures today, I, too, couldn't help but think maybe it was finally time to take Travis out back. What with the reduced capacity and impending `travis-ci.org` shutdown (open source projects will continue to build, just on a `travis-ci.com` instance), now's probably as good a time as any. I haven't been a huge fan of how travis has been going since they were purchased. The thing that keeps getting me, though, is I know the github actions and ADO are hosted on essentially the same infra (Azure), so could have correlated outages - and outage avoidance (so we could still get some PR statuses when there was a cloud outage of some sort) was one of the only reasons we were using multiple providers to begin with. By dropping travis, we'll finally concede that we don't _really_ care about that kinda uptime anymore. 🤷

File: src/testRunner/runner.ts [hunk 2]
```diff
@@ -209,12 +209,6 @@ namespace Harness {
             runners.push(new FourSlashRunner(FourSlash.FourSlashTestType.ShimsWithPreprocess));
             runners.push(new FourSlashRunner(FourSlash.FourSlashTestType.Server));
             // runners.push(new GeneratedFourslashRunner());
```
Comment (andrewbranch) [inline src/testRunner/runner.ts:211]:
```suggestion
            // runners.push(new GeneratedFourslashRunner());

            // CRON-only tests
            if (process.env.TRAVIS_EVENT_TYPE === "cron") {
                runners.push(new UserCodeRunner());
                runners.push(new DockerfileRunner());
             }
```

File: src/testRunner/runner.ts [hunk 3]
```diff
@@ -214,7 +214,7 @@ namespace Harness {
             if (process.env.TRAVIS_EVENT_TYPE === "cron") {
                 runners.push(new UserCodeRunner());
                 runners.push(new DockerfileRunner());
-            }
+             }
```
Comment (andrewbranch) [inline src/testRunner/runner.ts:217]:
```suggestion
            }
```

**Pull request merged**